src/github.ts

```diff
@@ -328,6 +327,8 @@ export class GitHub {
           error: logger.error,
         },
         retry: {
+          retries: options.retries ?? 0,
+          retryAfter: 3, // 3 seconds
           doNotRetry: [
             '403', // Used by GitHub when throttling
```

Comment:
Ooh, that's interesting. 403's where the error we were seeing. Why github abuses this error code is beyond me, but oh well. So we do need to retry on 403 (but slowly)

Reply:
It's a bit weird, but we are using two octokit plugins (that's recommended by their documentation). One to retry general errors, and one to handle throttling. I added 403 to the list on "do not retry" just to be sure the `retry` plugin doesn't handle retry for cases where the `throttling` plugin should.